components/script/script_thread.rs

```diff
@@ -2498,10 +2498,18 @@ impl ScriptThread {
             .find_document(parent_pipeline_id)
             .unwrap();
 
-        let iframes = document.iframes();
-        if let Some(iframe) = iframes.get(browsing_context_id) {
-            document.request_focus(Some(iframe.element.upcast()), FocusType::Parent, can_gc);
-        }
+        let Some(iframe_element_root) = ({
+            // Enclose `iframes()` call and create a new root to avoid retaining
+            // borrow.
+            let iframes = document.iframes();
+            iframes
+                .get(browsing_context_id)
+                .map(|iframe| DomRoot::from_ref(iframe.element.upcast()))
+        }) else {
```

Comment:
I think we can remove parenthesis:

```suggestion
        let Some(iframe_element_root) = {
            // Enclose `iframes()` call and create a new root to avoid retaining
            // borrow.
            let iframes = document.iframes();
            iframes
                .get(browsing_context_id)
                .map(|iframe| DomRoot::from_ref(iframe.element.upcast()))
        } else {
```

Reply:
The parentheses were initially added because without them, I get a build error: "error: right curly brace `}` before `else` in a `let...else` statement not allowed" with suggested fix of "wrap the expression in parentheses".

Reply:
Right.